Do not override CFLAGS/CPPFLAGS if they where set in the environment

--- Init.make.in~	2012-03-16 18:04:04.000000000 +0100
+++ Init.make.in	2013-01-25 11:22:48.341709373 +0100
@@ -45,8 +45,8 @@
 CC=		@CC@
 MKDEP=		@MKDEP@
 
-CPPFLAGS= 	-I$(GDHE_CLIENT_SRC) -I$(GDHE_TOGL_SRC) -I$(GDHE_TOGL_BUILD)
-CFLAGS=		@TCL_EXTRA_CFLAGS@
+CPPFLAGS+=	-I$(GDHE_CLIENT_SRC) -I$(GDHE_TOGL_SRC) -I$(GDHE_TOGL_BUILD)
+CFLAGS+=	@TCL_EXTRA_CFLAGS@
 
 
 LIBTOOL=	@LIBTOOL@
